src/main/java/lotto/domain/WinningInfo.java

```diff
@@ -11,16 +11,24 @@ private WinningInfo(Map<Rank, Integer> winningInfo) {
         this.winningInfo = winningInfo;
     }
 
-    public static WinningInfo of(PickedLottoNumbers pickedLottoNumbers, LottoNumbers winningNumbers) {
+    public static WinningInfo of(Lottos lottos, LottoNumber bonusNumber, Lotto winningNumbers) {
```

Comment:
winningNumbers도 Lotto인데 numbers로 파라미터 네이밍이 되어 헷갈리는 것 같습니다.

```suggestion
    public static WinningInfo of(Lottos lottos, LottoNumber bonusNumber, Lotto winningLotto) {
```